### PR TITLE
feat: add seq2seq task types

### PR DESCRIPTION
**Pull request by elboy3**
Add `seq2seq_completion` and `seq2seq_chat` to task types

### CODE REVIEW

File: dataquality/loggers/data_logger/__init__.py [hunk 1]
```diff
@@ -19,5 +23,7 @@
     "text_ner",
     "object_detection",
     "BaseGalileoDataLogger",
-    "seq2seq",
+    "base",  # Import needed for seq2seq subclass discovery
```
Comment (setu4993) [inline dataquality/loggers/data_logger/__init__.py:26]: Couldn't we just import directly in those files? Feels a bit weird to have `base` being exported.

Comment (setu4993) [reply dataquality/loggers/data_logger/__init__.py:26]: If we do need it, maybe makes sense to call it `seq2seq_base`? Just at the import-level would be fine IMO:
```suggestion
from dataquality.loggers.data_logger.seq2seq import base as seq2seq_base
...
    "seq2seq_base",  # Import needed for seq2seq subclass discovery
```

**Review by setu4993 — APPROVED** [review]
Broadly looks good, left 1 note about export name.

File: dataquality/loggers/data_logger/__init__.py (hunk 1 above)
Comment (elboy3) [reply dataquality/loggers/data_logger/__init__.py:26]: Sounds good, i'll rename to base!

**Pull request merged**